src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
```

Comment:
at the end of all comments, u will understand that u do not need this variable, because u will return the result immediately after the condition in your `if`